### PR TITLE
 Allow specifying var_file parameter for tasks

### PR DESCRIPTION
**Pull request by vlad-ro**
Initialize configuration_directory before calling lambdas to allow them manipulating the initialized state
Require ruby-terraform >= 0.11.2 for validate task
Fix homepage link

### CODE REVIEW

**Comment by vlad-ro** [comment]
Changing .ruby-version is not required, feel free to not merge this file.

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.0.0-p648
\ No newline at end of file
+2.5.1
```
Comment (tobyclemson) [inline .ruby-version]: Would you be able to revert this change? I want to test on the earliest supported Ruby which is currently 2.0.0 although may update to 2.3.7 soon.

Comment (tobyclemson) [reply .ruby-version]: Ah, sorry, missed your comment.

**Comment by tobyclemson** [comment]
This PR is great, thanks for getting it finished. I'll merge in and release now.

**Comment by tobyclemson** [comment]
@vlad-ro This is available in [0.19.0](https://rubygems.org/gems/rake_terraform/versions/0.19.0)
